scss/partials/_leadership-bio.scss

```diff
@@ -0,0 +1,127 @@
+/** TABLE OF CONTENTS
+ — — — — — — — — — — — — — — — — — — — — — — — — — — — — — — /
+
+  0 - VARIABLES
+  1 — Leadership Bio 
+  1.1 - Header
+  1.2 - Content
+  1.3 - Contact
+/// END TABLE OF CONTENTS **/
+
+// NOTE: Due to lack of mobile designs, I made the the decision to slace the padding/margin down to 1/2 for tablet and 1/3 for mobile.
+
+/** 0 - VARIABLES
+------------------------------------------------------------*/
+// TODO: Add variables for the bio.
+
+/** 1 - Leadership Bio
+------------------------------------------------------------*/
+.leadership-bio {
+  /* =========================
+1.1 - Header 
+========================= */
+  &__header {
+    background-color: $primary-blue;
+    color: $primary-white;
+    padding: 7.5rem 2rem 3.6rem 2rem;
+    margin-bottom: 5rem;
+
+    @include breakpoint(desktop) {
+      padding: 14.4rem 14.5rem 18.5rem 14.5rem;
+      margin-bottom: 14.5rem;
+    }
+  }
+
+  &__heading-main {
+    @extend h1;
+    padding-bottom: 2rem;
+  } 
+
+  &__subheading {
+    @extend .p--lead-1;
+  }
+  /* =========================
+1.2 - Content
+========================= */
+
+  &__content {
+    padding: 0 6rem 4.3rem 6rem;
+
+    @include breakpoint(desktop) {
+      padding: 0 18rem 5rem 18rem;
+    }
+  }
+
+  &__image {
+    margin-bottom: 2rem;
+
+    @include breakpoint(desktop) {
+      margin-bottom: 8.7rem;
+    }
+  }
+
+  &__heading--secondary {
+    @extend .h4;
+    margin-bottom: 2.4rem;
+  }
+
+  &__copy {
+    @extend .p--body;
+    color: $cool-neutral-1;
+    line-height: 2.8;
+  }
+  
+  /* =========================
+1.3 - Contact
+========================= */
+  &__contact {
+    padding: 0 6rem 5rem 6rem;
+
+    @include breakpoint(desktop) {
+      padding: 0 18rem 5rem 18rem;
+    }
+  }
+
+  &__heading-tertiary {
+    @extend [class^="tertiary-info"];
+    margin-bottom: 2.7rem;
+  }
+
+  &__social-icons {
+    display: flex;
+    flex-direction: row;
+    padding-left: 0;
+  }
+  &__social-icon {
```

Comment:
At some point, we should make `__social-icon` in a mixin or a utility since the same rules and declarations are happing in the footer sass file.